code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -2014,6 +2014,20 @@
 						dab_id?.brain_damage_count += 10
 						if(src.get_brain_damage() > 60)
 							src.show_text(__red("Your head hurts!"))
+					for(var/atom/A as anything in src.loc)
+						if(istype(A, /obj/item/storage/bible))
```

Comment:
Actually this could just be a `if(locate(/obj/item/storage/bible) in src.loc)` and likely faster.

Reply:
added